docker/dev-ubuntu/Dockerfile

```diff
@@ -13,6 +13,12 @@ RUN apt-get update -qq && apt-get install -y --no-install-recommends \
 #Switch back to the normal user
 USER opam
 
```

Comment:
Probably this folder should be renamed to debian at some point.

Reply:
Done.